tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -52,6 +52,7 @@ Object.Pipeline {
 			Object.Widget.dai-copier.1 {
 				stream_name $SDW_SPK_STREAM
 				dai_type "ALH"
+				dai_index	20
```

Comment:
here too. I must admit I don't follow what this dai_index is. It clearly makes no sense if this is used to name a DAI.

Reply:
> here too. I must admit I don't follow what this dai_index is. It clearly makes no sense if this is used to name a DAI.

`20` means pcm `2`, index `0`.
We include pcm id in the dai_index because the dais are defined in different files and sdw-xxx-generic.conf can be included by different sdw topologys. The dai index could be conflict if they are 0,1,2,3... We don't know what index can we use when we add a new sdw topology unless we check all included .confs.
For the reason above, I think use different dai index group for differnt pcm could be easier to decide a new dai index.